optaplanner-quickstarts/quarkus-facility-location/src/main/java/org/acme/facilitylocation/bootstrap/RepositoryPopulator.java

```diff
@@ -0,0 +1,34 @@
+package org.acme.facilitylocation.bootstrap;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Observes;
+
+import org.acme.facilitylocation.domain.FacilityLocationProblem;
+import org.acme.facilitylocation.domain.Location;
+import org.acme.facilitylocation.persistence.FacilityLocationProblemRepository;
+
+import io.quarkus.runtime.StartupEvent;
+
+@ApplicationScoped
+public class RepositoryPopulator {
```

Comment:
Seems to do the same thing as DemoDataGenerator in the timetabling example.
We should try to use the same naming then

Reply:
the fact that internally it does it in a very different way (not filling a database, but a in memory store), is irrelevant for the name consistency I think: we want users to easily switch between examples.